server/scratchy_server/model/messageModel.py

```diff
@@ -5,14 +5,17 @@
 
 
 class MessageModel(db_scratchy.Document):
-    content = db_scratchy.StringField()
-    author = db_scratchy.ObjectIdField()
+    content = db_scratchy.StringField(required=True)
+    author = db_scratchy.ObjectIdField(required=True)
     timestamp = db_scratchy.DateTimeField(default=datetime.datetime.utcnow)
-    roomId = db_scratchy.ObjectIdField()
+    roomId = db_scratchy.ObjectIdField(required=True)
 
 
 class MessageSchema(ma.Schema):
     id = ObjectIdSchema()
-    content = ma.fields.Str()
-    author = ma.fields.Str()
-    roomId = ma.fields.Str()
+    content = ma.fields.Str(required=True)
+    author = ma.fields.Str(required=True)
+    roomId = ma.fields.Str(required=True)
+    # timestamp is unuse for now
+    # because date seem not compatible with marshmallow datetime
+    # it will be added with marshallow mongo update
```

Comment:
Thanks for this comment, i think datetime should be set only by the server itself so it's good for me